contentcuration/contentcuration/assets/channelList/vuex/channelList/index.js

```diff
@@ -0,0 +1,14 @@
+import * as getters from './getters';
+import * as mutations from './mutations';
+import * as actions from './actions';
+
+export default {
+  namespaced: true,
+  state: {
+    channelsMap: {},
```

Comment:
This is nice and simple, but it would be nice to be able to define a schema for what the channel objects should look like.